app/routes.py

```
@@ -205,15 +205,13 @@ async def join_meeting(request: BotRequest, client_request: Request):
-    MEETING_DETAILS[bot_client_id] = {
-        "meeting_url": request.meeting_url,
-        "persona_name": resolved_persona_data.get("name", persona_name_for_logging), # Use display name from resolved data
-        "meetingbaas_bot_id": None,  # Will be set after creation
-        "enable_tools": request.enable_tools,
-        "streaming_audio_frequency": streaming_audio_frequency,
-        "final_prompt": final_prompt, # This is the prompt with interaction instructions
-        "persona_data": resolved_persona_data # Store the resolved persona data
-    }
+    MEETING_DETAILS[bot_client_id] = (
+        request.meeting_url,
+        resolved_persona_data.get("name", persona_name_for_logging),  # Use display name from resolved data
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)